database/src/main/resources/db/migration/V50__saksnummer.sql

```diff
@@ -0,0 +1 @@
+alter table sak add column saksnummer int generated by default as identity (start with 2021)
```

Comment:
Gjør identity at den lager en unik index?

Reply:
Purrer på denne :)

Reply:
Generated as identity, är ganska likt som med att använda `SERIAL` men den är nyare har en del fördelar, exempelvis bättre kompatibilitet med andra databaser (Man kan byta mellan Postgresql, DB2, Oracle utan problem).

Varje insert ska ju ha ett nytt värde, men inte säker hur det blir om man får in två samtidigt. Kanske kan lägga till en UNIQUE constraint eller noe? Tror inte den är unik out of the box.

Reply:
Jeg stemmer for en UNIQUE constraint!